frontend/src/concepts/pipelines/content/createRun/contentSections/ProjectAndExperimentSection.tsx

```diff
@@ -39,7 +44,11 @@ const ProjectAndExperimentSection: React.FC<ExperimentSectionProps> = ({
         <FormGroup label="Experiment" aria-label="Experiment" isRequired>
           <Stack hasGutter>
             <StackItem>
-              <ExperimentSelector selection={value?.display_name} onSelect={onChange} />
+              {isSchedule ? (
+                <AllExperimentSelector selection={value?.display_name} onSelect={onChange} />
```

Comment:
why would we show all experiments here?

Reply:
If it's on the create schedules page, we allow the users to create a schedule with an archived experiment, but we will set the schedule to disabled status instead of enabling it. I found this on the mockup.https://www.figma.com/proto/etbN0ccCcZR1qKCpJXQNAg/Pipeline-phase-2?page-id=615%3A34927&type=design&node-id=2005-154680&viewport=401%2C-96%2C0.02&t=hix7x325vWWGWrfX-1&scaling=scale-down-width&starting-point-node-id=2005%3A151595

Reply:
interesting. ok